news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -26,96 +27,98 @@ First of all, congratulations to the winners of the April 2017 Spotlights. They
 
 [![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/re-lm.jpg)](https://osu.ppy.sh/s/588066)
 
-This map caught my eye for a few reasons. The most obvious being the heavy use of SV, which is thoughtfully crafted to be intuitive and smooth, which is quite impressive given how easy it is to cross the line of being more of a distraction than a feature. Though the SV is not the main reason I think this map is particularly deserving of a spotlight. Lude doesn't shy away from LN usage in their mapping of this track. Instead, they use it to add atmospheric value, complexity and originality, which is what I personally love to see. These two points combined with some enjoyable jack-oriented mapping and very smooth difficulty curves, makes this a suitable pick for May's spotlight.
+This map is eye-catching for a few reasons: The most obvious being the heavy use of scroll speed variations, which are thoughtfully crafted to be intuitive and smooth. This is quite impressive given how easy it is to cross the line of being more of a distraction than a feature. However, the scroll speed variations are not the main reason this map is particularly deserving of a spotlight. **[Lude](https://osu.ppy.sh/users/6974536)** doesn't shy away from long note usage in their mapping of this track. Instead, they use it to add atmospheric value, complexity, and originality, which is great to see in the ranked section. These two points combined with some enjoyable jack-oriented mapping and very smooth difficulty curves makes this a suitable pick for May's spotlight.
 
 [![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/black-or-white.jpg)](https://osu.ppy.sh/s/565779)
 
-Let me introduce you another extreme 7K marathon map by Kawawa. I really like how the map structure flows nicely with the song intensity. From the map itself, i can see the mapper have a really good experience in term of emphasizing the melody itself. What caught my attention is the jacking style pattern in the map which it is pretty new and interesting to see. Besides giving a great pattern usage, the speed change through the map are astonishing extreme and fun. The mapper really did a good job on hightlighting certain beats magnitude with a suitable slider velocity. By advancing through the map, player could easily experience the consciousness of the music along with the note arrangement emphasized by the mapper intention. Have a try while you are on it.
+Let me introduce you another extreme 7K marathon map by **[Kawawa](https://osu.ppy.sh/users/4647754)**. Pleasure to the eye how the map structure flows nicely with the song intensity. He has a really good experience in term of emphasizing the melody itself. What caught my attention is the jacking style pattern in the map which it is pretty new and interesting to see. Besides giving a great pattern usage, the speed change through the map are astonishing extreme and fun.
```

Comment:
Flows really awkwardly compared to the rest of the article, needs a full rewrite.

Kawawa takes a spot in this month's charts with his extreme 7K marathon rendition of Black or White. Pleasing to the eye and full of well-designed flow, he managed to craft the map masterfully around the music, emphasizing the very melody of the track itself. An all-new style of jackhammer patterning adds a bit of flair to the map, accentuated further by some extremely well planned speed changes to spice things up.

An excellent mapset all around, we're confident you'll love Black and White particularly in regards to how close it follows the very consciousness and arrangement of the music. Give it a go!